packages/webarchitect/README.md

```diff
@@ -0,0 +1,251 @@
+
+You can create custom rulesets by following the JSON Schema format used by the built-in examples. The [JSON Schema documentation](https://json-schema.org/learn/) provides comprehensive guidance on writing schema definitions. Custom rulesets can extend existing ones or define completely new validation requirements for your specific organizational needs.
+
+This approach gives you the full power of JSON Schema's validation capabilities while maintaining a consistent structure for how validation rules are defined and applied across different projects and teams.
+
+## CLI API
+
+Webarchitect provides a simple but powerful command-line interface designed to integrate easily into development workflows and CI/CD pipelines. The tool must be run from the top-level directory of your package - the same directory that contains your package.json file - to properly validate your project structure.
+
+### Basic Command Structure
+
+```bash
+webarchitect <ruleset> [options]
+```
+
+### Available Options
+
+- `--verbose`: Shows detailed information about each validation, including what files are being checked, what rules are being applied, and the actual content found. Essential for debugging validation failures or understanding exactly what the tool is checking.
+-`--json`: Outputs results in JSON format instead of human-readable text. Required for integrating webarchitect into automated systems, CI/CD pipelines, or other tools that need to programmatically process validation results.
+-`--help`: Displays comprehensive usage information, available options, and examples. Use this command to see the most current information about the tool's capabilities.
+
+### Ruleset Resolution Mechanism
+
+The `<ruleset>` parameter follows a specific resolution order that determines where webarchitect looks for ruleset definitions:
+
+1. **Local Files First** - Webarchitect first checks the current directory for a file matching your ruleset name. If you specify `my-rules`, it looks for `my-rules.json` in the current directory. This allows you to override built-in rulesets or use project-specific validation rules.
+
+2. **Bundled Rulesets Second** - If no local file is found, webarchitect searches its internal collection of built-in rulesets. These include `base`, `base-package`, and `package-react`. Built-in rulesets are distributed with the tool and provide standard validation patterns for common project types.
+
+3. **Remote URLs** - You can also specify a complete URL (starting with `http://` or `https://`) to fetch rulesets from remote locations. Remote rulesets must end with `.json` and be accessible via standard HTTP requests. This enables sharing rulesets across organizations or using centrally managed validation rules.
```

Comment:
Do you plan to account for remote rulesets that are behind an auth requirement?

Reply:
It's 100% out of scope right now - just picking your brain. My mind went there because of the wording "standard HTTP requests" - i wondered what "standard" meant.